pisa-proxy/parser/mysql/src/grammar.y

```diff
@@ -7302,6 +7353,488 @@ alter_lock_option_value -> String:
       }
 ;
 
+create_user_list -> Vec<UserWithAuthOption>:
+      create_user
+      {
+           vec![$1]
+      }
+    | create_user_list ',' create_user
+      {
+           $1.push($3);
+           $1
+      }
+;
+
+create_user -> UserWithAuthOption:
+      user identification opt_create_user_with_mfa
+      {
+          UserWithAuthOption::UserIdentification(UserIdentification {
+              span: $span,
+              user: $1,
+              identification: $2,
+              opt_create_user_with_mfa: $3,
+          })
+      }
+    | user identified_with_plugin opt_initial_auth
+      {
+           UserWithAuthOption::UserIdentifiedWithPlugin(UserIdentifiedWithPlugin {
+               span: $span,
+               user: $1,
+               identified_with_plugin: $2,
+               opt_initial_auth: $3,
+           })
+      }
```

Comment:
There is conflict for `user identification opt_create_user_with_mfa` and `user identified_with_plugin opt_initial_auth`.

If remove the `user identified_with_plugin opt_initial_auth`, the test case at `parser#182` is OK.

and how to fix it.